### PR TITLE
User can see activities on map

### PR DESCRIPTION
**Pull request by sundqvistjohan**
[PT Link](https://www.pivotaltracker.com/story/show/171233247)

As a user
In order to find my way around on my trip
I want the trips activities, restaurants and hotel visible on a map

Adds Result component where final trip is shown
Adds map with activities (screenshot is a mock-up with a lot of different activity types)
Moves ActivitiesList to Result component
Screenshot 2020-02-12 at 16 18 58

### CODE REVIEW

File: src/components/ResultMap.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+const ResultMap = props => {
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  const [activeMarker, setActiveMarker] = useState({});
+  const [selectedPlaces, setSelectedPlaces] = useState({});
+
+  let activityTypes;
+  let marker
+  let activityMarkers = [];
+
+  useEffect(() => {}, [props.activities]);
```
Comment (mantanquzzmo) [inline src/components/ResultMap.jsx]: what does one do?

File: src/components/ResultMap.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    setSelectedPlaces(props)
+    setActiveMarker(marker)
+    setShowInfoWindow(true)
+  };
+
+  if (props.activities) {
+    activityTypes = Object.keys(props.activities);
+
+    activityTypes.map(activityType => {
+      debugger
```
Comment (mantanquzzmo) [inline src/components/ResultMap.jsx]: remove

File: src/components/ResultMap.jsx (hunk 1 above)
Comment (sundqvistjohan) [reply src/components/ResultMap.jsx]: Ah, nothing anymore after setActivities was moved to another component. Removed.

File: src/components/ResultMap.jsx (hunk 2 above)
Comment (sundqvistjohan) [reply src/components/ResultMap.jsx]: done

**Pull request merged**